.circleci/create_circleci_config.py

```diff
@@ -503,6 +503,7 @@ def job_name(self):
         "pip install -U --upgrade-strategy eager git+https://github.com/huggingface/accelerate",
         "pip install --upgrade --upgrade-strategy eager pytest pytest-sugar",
         "pip install -U --upgrade-strategy eager natten",
+        "pip install -U --upgrade-strategy eager einops",
```

Comment:
no longer to have this here, as we don't run the new file in CircleCI, and it looks like we already have this in the docker image for daily CI.